charts/router/templates/router-deployment.yaml

```diff
@@ -76,3 +77,4 @@ spec:
             port: 9090
           initialDelaySeconds: 1
           timeoutSeconds: 1
+{{ end }}
```

Comment:
These files should end with a newline to satisfy some tools.

Would it improve readability to finish with a comment like:
```yaml
{{ end }}{{/* if not .Values.global.experimental_native_ingress */}}
```
I'm just asking--not sure myself.